packages/common/src/chains/sepolia.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "sepolia",
+  "chainId": 11155111,
+  "networkId": 11155111,
+  "defaultHardfork": "istanbul",
+  "consensus": {
+    "type": "pow",
+    "algorithm": "ethash",
+    "ethash": {}
+  },
+  "comment": "PoW test network",
```

Comment:
Something more specific would be nice here "PoW test network to replace Ropsten" e.g..